src/cli/commands.ts

```diff
@@ -66,7 +67,12 @@ cli
         manifestVersion: flags.mv3 ? 3 : flags.mv2 ? 2 : undefined,
         configFile: flags.config,
         debug: flags.debug,
-        analysis: flags.analyze ? { enabled: true } : undefined,
+        analysis: flags.analyze
+          ? {
+              enabled: true,
+              open: flags.analyzeOpen,
+            }
+          : undefined,
```

Comment:
@aklinker1 how hard would it be to respect `analysis.open` from the config file if `--analyzeOpen` isn't passed?

Reply:
It should already behave that way. The CLI flags override the relevant options in the config file

I would recommend setting the value in the config file if you always want to open it.

Reply:
@aklinker1 this is not how things are behaving for me locally (on 0.17.8) - to clarify, my config looks like this:

```
analysis: { outputFile: "build/stats.html", open: true }
```

and when I run `wxt build --analyze`, `analysis.outputFile` is respected, but `analysis.open` is not.

FYI, I don't specify `analysis.enable` in my config as I only want to do this in a specific NPM script.

Could it be that the override logic isn't taking into account that `flags.analyzeOpen` is `undefined`, but `analysis.open` still exists if `flags.analyze` does (like how `"prop" in { prop: undefined }` is `true`)?

Reply:
Interesting, let me take a look. You're setup is what I intended, so I'll fix it

Reply:
Yup, the `undefined` was overriding the `true`:

<img width="512" alt="Screenshot 2024-03-25 at 12 11 08 AM" src="https://github.com/wxt-dev/wxt/assets/10101283/59d1fbbc-1f03-463a-9e00-d0a8a6894baf">

> `userConfig` comes from the wxt.config.ts file, `inlineConfig` is the CLI flags, and merged is the output of merging them together.

Caused by the spread operator, replaced with `defu` and it's working: https://github.com/wxt-dev/wxt/pull/567

Reply:
@joealden Released fix in [v0.17.9](https://github.com/wxt-dev/wxt/releases/tag/v0.17.9)